### DISCUSSION TITLE
Posy's Improved Cursors

### DISCUSSION THREAD

**Discussion by MrDowntempo** [catppuccin/catppuccin#2081] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

YouTuber Posy got frustrated with the default Windows cursors and made his own, which are a big improvement, and also include some rainbow elements. They'd look great in Catppuccin colors. He has [a video](https://www.youtube.com/watch?v=YThelfB2fvg) describing why and how. He also has a [link](http://www.michieldb.nl/other/cursors/) where he makes all the cursors available for free. Although it isn't entirely clear how permissive his license is. He mentions he designed the cursors in Photoshop, but used [RealWorld Cursor Editor](http://www.rw-designer.com/cursor-maker) to turn them into cursors. There is a [linux fork here](https://github.com/simtrami/posy-improved-cursor-linux)

### Any additional comments?

I've been tinkering with the cursor editor but haven't had any luck yet it getting the aliased colors in correctly. This might be beyond my skillset. Willing to help out in any way I can, but probably can't tackle this alone. 

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by nekowinston** [comment]
Hey there, I've taken a look at this, and the licensing might be an issue.

Neither the Windows version nor the Linux fork specify a License, so we'd have to reach out to the author to create a fork with Catppuccin colors.

**Reply by MrDowntempo** [reply]
Yeah, that certainly looks to be the case. I did post a YouTube comment asking for clarification, but its an old video and probably hasn't been seen. I could write up an e-mail and reach out to him. He has some contact info on his webpage.

**Reply by nekowinston** [reply]
I'd appreciate that, but I imagine people reached out before. It might still be worth a shot.
Unless this licensing issue is resolved, we won't be able to adopt it as an official port.

**Reply by MrDowntempo** [reply]
Ok, I'll report back here if and when I get a response

**Comment by MrDowntempo** [comment]
I reached out to him via his e-maill and he did get back to me with permission. My attempts at actually achieving decent results have so far been a failure. (My very first time experimenting with cursors and the software to create them.) Anyways. I'm not sure the best way to provide that permission here. I have the e-mails, and can copypasta in here I guess since his e-mail is public.

> Op zo 11 jun 2023 om 02:37 schreef <[coreywoodworth@gmail.com](mailto:coreywoodworth@gmail.com)>:
> Hi Posy!
> I’ve been watching your videos for a little while, but only recently discovered your older video where you designed new cursors. I really love them, and have been using them on my Windows install ever since. I’ve also recently found the Catppuccin theme, which is a collection of themes for operating systems and applications that all match one of 4 predefined pallets, to give a desktop a very uniform pastel rainbow look. They have a github page: https://github.com/catppuccin
>  
> I would really like to use your cursors with that theme, but recolored to match. And Ideally, I would like to make those changes available to the rest of the Catppuccin community. However, while I see you’ve made your cursors free to use, I couldn’t find any information on your page or in the downloads about how you might have licensed them. Are they just public domain? One of the Creative Commons licenses? Something else? Or would you simply prefer no derivative works or ports be made? Catppuccin mostly uses the MIT license, but I believe there’s precedent for making exceptions when necessary.
>  
> I’ve brought this up on their Discord, and submitted a request on their Github. https://github.com/catppuccin/catppuccin/discussions/2081 There is some interest from others in the community. But of course, we want to make sure we do right by you, the original creator of the cursors. Can you let us know if there’s an explicit license we should follow, or if this is just out of the cards?
>  
> Thanks,
> Corey (MrDowntempo)
>  
> From: Michiel de Boer <[michiel@michieldb.nl](mailto:michiel@michieldb.nl)> 
> Sent: Sunday, June 11, 2023 4:43 AM
> To: [coreywoodworth@gmail.com](mailto:coreywoodworth@gmail.com)
> Subject: Re: Info about License for Your Improved Cursors?
>  
> Thank you! I haven't put any license on it that's correct so in theory they're copyrighted. However, now that I write that you can use them for the project, you are now sure you're allowed to use them ;-)
>  
> A question from my side: Is that theme for various operating systems? The page doesn't say.
>  
> Cheers,
> Michiel
>  
> Op zo 11 jun 2023 om 11:18 schreef <[coreywoodworth@gmail.com](mailto:coreywoodworth@gmail.com)>:
> Great thanks! I’ll pass that info along to the team.
>  
> So I’m still pretty new with Catppuccin, and not an authority on it. But its mostly done on a per app basis. They split their packages up into what they call ‘ports’ So there’s ports for many things like VIM, Firefox, Discord, Powershell, Notepad++, Steam etc. But also some larger ports for things like desktop environments such as KDE and toolkits like GTK etc. So while there may not be a single “Windows 10” or “OSX” package, there’s enough individual ports to theme your full desktop regardless of your OS. The ports on the github are only the official ones too. On the discord you can find more ports that are either still in development or antiquated and not made official for one reason or another. Since their work is under MIT license, others have forked it and you can find some individuals making compatible wallpapers and themes for windows theme engines (I’ve not messed with those, myself) on places like DeviantArt and the [Catppuccin subreddit](https://www.reddit.com/r/catppuccin/) (which is going Dark in protest of the current reddit API drama).
>  
> Appreciate the permission and clarification, and thanks again for the lovely cursors. They’re such a simple, but definite improvement over the defaults.
>  
> Corey
